Cargo.toml

```diff
@@ -22,6 +22,7 @@ env_logger = "0.5.3"
 glob = { version = "0.2.11", optional = true }
 rayon = { version = "1.0", optional = true }
 clap-verbosity-flag = "0.1.0"
+tempfile = "3.0.3"
```

Comment:
Can you put this into a `[dev-dependencies]` section?

Reply:
Done!